R/plotLoadings.R

```diff
@@ -0,0 +1,374 @@
+#' Plot feature loadings after performing a reducedDim 
+#'
+#' Inspired by the \code{\link[diffTop:plotASVcircular]{plotASVcircular}} method using phyloseq 
+#' and has been converted to use TreeSummarizedExperiment objects.
+#'
+#' @param x a
+#'   \code{\link[TreeSummarizedExperiment:TreeSummarizedExperiment-class]{TreeSummarizedExperiment}}
+#'   x.
+#' 
+#' @param dimred One dimensionality reduction method of reducedDimNames
+#'   indicating which method will be used.
+#'   (default: \code{dimred = "PCA"})
```

Comment:
Put this parameter descriptions in this format.

Class. Definition. Default value.

-->

dimred \code{Character scalar}. DImension reduction from reducedDimNames to be plotted. (Default: \code{"PCA"])